src/otx/algo/segmentation/segnext.py

```diff
@@ -4,64 +4,124 @@
 """SegNext model implementations."""
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any, Literal
+from typing import TYPE_CHECKING, Any, ClassVar
 
-from otx.algo.utils.mmconfig import read_mmconfig
+from otx.algo.segmentation.backbones import MSCAN
+from otx.algo.segmentation.heads import LightHamHead
 from otx.algo.utils.support_otx_v1 import OTXv1Helper
-from otx.core.exporter.base import OTXModelExporter
-from otx.core.exporter.native import OTXNativeModelExporter
-from otx.core.metrics.dice import SegmCallable
-from otx.core.model.base import DefaultOptimizerCallable, DefaultSchedulerCallable
-from otx.core.model.segmentation import MMSegCompatibleModel
-from otx.core.schedulers import LRSchedulerListCallable
-from otx.core.types.label import LabelInfoTypes
-from otx.core.utils.utils import get_mean_std_from_data_processing
+from otx.core.model.segmentation import TorchVisionCompatibleModel
+
+from .base_model import BaseSegmNNModel
 
 if TYPE_CHECKING:
-    from lightning.pytorch.cli import LRSchedulerCallable, OptimizerCallable
+    from torch import nn
+
+
+class SegNextB(BaseSegmNNModel):
+    """SegNextB Model."""
+
+    default_backbone_configuration: ClassVar[dict[str, Any]] = {
```

Comment:
Is it required? They are already in the recipes.

Reply:
It is more for API usage. If we don't add this default - user should pass this entirely to initialize the class. With this addition, we can just create model passing only base_model_name. Moreover, we can pass only several parameters via dict (for backbone or head) and it will override defaults. I thought it can be useful.

Reply:
I agree with your intention. I have considered how to use it more cleanly, but I have no idea :)